roles/agent_install/tasks/main.yml

```diff
@@ -44,15 +44,15 @@
         group: root
         mode: 0644
 
-    - name: Enable eBPF
-      when: agent_install_driver_type == "ebpf"
+    - name: Enable Universal / Legacy eBPF
+      when: agent_install_driver_type == "universal_ebpf" or agent_install_driver_type == "legacy_ebpf"
       block:
-        - name: (eBPF) Enable eBPF probe
-          ansible.builtin.lineinfile:
-            path: /etc/sysconfig/dragent
-            regexp: SYSDIG_BPF_PROBE
-            line: export SYSDIG_BPF_PROBE=
-            create: true
+        - name: (eBPF) Enable Universal / Legacy eBPF driver
+          ansible.builtin.template:
+            src: dragent_ebpf_env_file.j2
+            dest: "/etc/{% if ansible_pkg_mgr == 'apt' %}default{% else %}sysconfig{% endif %}/dragent"
```

Comment:
@mprzybylski do you know why this file is indifferent location depending from the package manger? 🤔

Reply:
@mavimo, the way I understand it is different distribution families following slightly different layout conventions.